yml/dp02_dc2.yaml

```diff
@@ -7083,696 +7083,696 @@ tables:
   "@id": "#DiaObject"
   description: this is diaObjectTable_tract in the butler repo
   columns:
+  - name: diaObjectId
+    "@id": "#DiaObject.diaObjectId"
+    datatype: long
+    mysql:datatype: BIGINT
+    description: Unique id.
+  - name: ra
+    "@id": "#DiaObject.ra"
+    datatype: double
+    mysql:datatype: DOUBLE
+    description: RA-coordinate of the position of the object at time radecTai.
   - name: decl
     "@id": "#DiaObject.decl"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
-  - name: diaObjectId
-    "@id": "#DiaObject.diaObjectId"
+    description: Decl-coordinate of the position of the object at time radecTai
```

Comment:
Per [RFC-863](https://jira.lsstcorp.org/browse/RFC-863), can we start referring to these as "Right Ascension" and "Declination" in the descriptions, and not propagating the "decl" naming if we can help it?